pkg/blobstore/cloud/s3_copy_mutator.go

```diff
@@ -0,0 +1,34 @@
+package cloud
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+
```

Comment:
Nit: Remove this empty line. The current convention is to group imports of the same 2nd level domain together.

Reply:
Done